Code/GraphMol/Descriptors/AtomicEnvironmentVector.cpp

```diff
@@ -116,77 +116,77 @@ ArrayXXd IndexSelect(ArrayXXd vector1, ArrayXXd vector2, ArrayXd index,
 */
 ArrayXd NeighborPairs(ArrayXXd coordinates, VectorXd species, double cutoff,
                       int numAtoms) {
-  auto padding_mask = species.array() == -1;
+  auto paddingMask = species.array() == -1;
   int cols = 0;
-  MatrixXd upper_triag(2, numAtoms * (numAtoms - 1) / 2);
+  MatrixXd upperTriag(2, numAtoms * (numAtoms - 1) / 2);
   for (auto i = 0; i < numAtoms; i++) {
     for (auto j = i + 1; j < numAtoms; j++) {
-      upper_triag.col(cols) << i, j;
+      upperTriag.col(cols) << i, j;
       cols++;
     }
   }
 
-  ArrayXd upper_triag_flattened(numAtoms * (numAtoms - 1));
+  ArrayXd upperTriagFlattened(numAtoms * (numAtoms - 1));
   int index = 0;
-  for (auto i = 0; i < upper_triag.rows(); i++) {
-    for (auto j = 0; j < upper_triag.cols(); j++) {
-      upper_triag_flattened(index) = upper_triag(i, j);
+  for (auto i = 0; i < upperTriag.rows(); i++) {
+    for (auto j = 0; j < upperTriag.cols(); j++) {
+      upperTriagFlattened(index) = upperTriag(i, j);
       index++;
     }
   }
 
-  ArrayXXd pair_coordinates(numAtoms * (numAtoms - 1), 3);
-  pair_coordinates =
-      IndexSelect(pair_coordinates, coordinates, upper_triag_flattened, 0);
+  ArrayXXd pairCoordinates(numAtoms * (numAtoms - 1), 3);
+  pairCoordinates =
+      IndexSelect(pairCoordinates, coordinates, upperTriagFlattened, 0);
 
   ArrayXXd distances(numAtoms * (numAtoms - 1) / 2, 3);
-  int num_pairs = numAtoms * (numAtoms - 1) / 2;
+  int numPairs = numAtoms * (numAtoms - 1) / 2;
   for (auto i = 0; i < numAtoms * (numAtoms - 1) / 2; i++) {
     distances.row(i) =
-        pair_coordinates.row(i) - pair_coordinates.row(i + num_pairs);
+        pairCoordinates.row(i) - pairCoordinates.row(i + numPairs);
   }
 
-  auto distance_norm = distances.matrix().rowwise().norm().array();
+  auto distanceNorm = distances.matrix().rowwise().norm().array();
 
-  ArrayXXd pair_padded_mask(numAtoms * (numAtoms - 1), 1);
-  for (auto i = 0; i < upper_triag_flattened.size(); i++) {
-    pair_padded_mask(i, 0) = padding_mask(upper_triag_flattened(i), 0);
+  ArrayXXd pairPaddedMask(numAtoms * (numAtoms - 1), 1);
+  for (auto i = 0; i < upperTriagFlattened.size(); i++) {
+    pairPaddedMask(i, 0) = paddingMask(upperTriagFlattened(i), 0);
   }
 
-  ArrayXd dist(distance_norm.size());
+  ArrayXd dist(distanceNorm.size());
 
-  for (auto i = 0; i < distance_norm.size(); i++) {
-    if (pair_padded_mask(i, 0) == 1) {
+  for (auto i = 0; i < distanceNorm.size(); i++) {
+    if (pairPaddedMask(i, 0) == 1) {
       dist(i) = INFINITY;
     } else {
-      dist(i) = distance_norm(i);
+      dist(i) = distanceNorm(i);
     }
   }
 
   auto x = dist.array() <= cutoff;
 
   std::vector<int> indices;
-  std::vector<std::pair<int, int>> atom_index12;
+  std::vector<std::pair<int, int>> atomIndex12;
```

Comment:
In lot of declarations, you use this "12" is it necessary ?

Reply:
The "12" is just to show that each entry is basically a pair. In some cases, I've had to use Eigen arrays in that each column represents a pair. It can be replaced with "Pair".

Reply:
Pair make more sense to me yes